free-courses-en.md

```diff
@@ -625,6 +625,7 @@
 
 ### Web Development
 
+* [almanac.httparchive.org](https://almanac.httparchive.org/) -HTTP Archive's annual "State of the Web" report made by the web community.
```

Comment:
Could you please stick to the naming conventions used in the repository?
{Title} - {Author}

```suggestion
* [Web Almanac](https://almanac.httparchive.org/) - HTTP Archive
```

Reply:
Sure!